src/vs/workbench/api/common/extHostTypes.ts

```diff
@@ -4608,30 +4608,34 @@ export class LanguageModelChatMessage implements vscode.LanguageModelChatMessage
 
 	role: vscode.LanguageModelChatMessageRole;
 
-	// Temp to avoid breaking changes
-	content2: (string | LanguageModelToolResultPart | LanguageModelToolCallPart)[] | undefined;
+	content: (LanguageModelTextPart | LanguageModelToolResultPart | LanguageModelToolCallPart)[];
```

Comment:
It's possible that this breaks really old proposal users who still assume that `content` can be a string, but this seems fair.

Reply:
Yeah, it's even a breaking change wrt the stable API but we felt lucky because the type is meant to be just passed on. Tho, there might be cases in which extension do `msg.content = 'hello'`

@roblourens maybe we should be graceful for that case and still support this